cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
```

Comment:
The `validate_ticket` method is being called with `attrs['row']` and `attrs['seat']`. Ensure these keys exist in `attrs` to avoid a KeyError. Consider using `attrs.get('row')` and `attrs.get('seat')`.